deployment/HDI-Spark/Notebooks/install-sparkdotnet-wn.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Can we combine both scripts into one script ?

Reply:
`install-livy-sparkmagic-hn.sh` is to submit script action on head node. And `install-sparkdotnet-wn.sh` is for worker node. That's why we keep two separate scripts.

Reply:
To make it easier on the user, I think it would be better if you can combine the two scripts and the user can just use the same script for both the headnode and workernode.

Reply:
Make sense, I will try to combine both into one and make user experience easier. Thanks for the suggestion!

Reply:
Done!